homeassistant/components/zha/fan.py

```diff
@@ -181,6 +180,8 @@ async def _async_set_fan_mode(self, fan_mode: int) -> None:
 class FanGroup(BaseFan, ZhaGroupEntity):
     """Representation of a fan group."""
 
+    _attr_translation_key: str = "fan_group"
```

Comment:
And why did this gain one?

Reply:
Imho a group a fans is different from a single fan. This was odd before, seems the group would done some weird device name default.